sdks/python/setup.py

```diff
@@ -89,7 +89,7 @@ def get_version():
     'google-apitools>=0.5.2',
     'httplib2>=0.8',
     'mock>=1.0.1',
-    'oauth2client>=2.0.1,<4.0.0',
+    'oauth2client>=2.0.1,<4.0.0dev',
```

Comment:
In which case <4.0.0dev wouldn't fix this either. Prereleases are handled the way they are specifically so that you can write something like <4.0.0 and have it do the right thing.